### ISSUE TITLE
[groheondus] Inconsistent bridge parameters definition

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-addons#10411]
The two parameters of the bridge are set as required and not required at the same time; it is inconsistent.

```
	<bridge-type id="account">
		<label>GROHE ONDUS Account</label>
		<description>This is an interface to the GROHE ONDUS Account as it is used by the app. If username and password are
			not set, you can configure to use a `refreshToken` to login. Read the README to get more info.</description>

		<config-description>
			<parameter name="username" type="text" required="false">
				<label>Username</label>
				<description>Username as used in the GROHE ONDUS App, usually your e-mail address.</description>
				<required>true</required>
			</parameter>
			<parameter name="password" type="text" required="false">
				<label>Password</label>
				<required>true</required>
				<context>password</context>
				<description>Password as used in the GROHE ONDUS App.</description>
			</parameter>
		</config-description>
	</bridge-type>
```

What is the good value ? I assume they are required ?

**Comment by lolodomo** [comment]
@FlorianSW : can you please answer ?

**Comment by FlorianSW** [comment]
Oha, good catch, that doesn't make much sense. The `username` and `password` fields are conditional and therefore not required (you can use them or instead use the `refreshToken` mechanism).

**Comment by FlorianSW** [comment]
See #10416

**Comment by lolodomo** [comment]
Closed by #10416
